### DISCUSSION TITLE
v2.1.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#3737] [Releases & Announcements]
## Changelog

- Ability to disable messages in resolved Live chat conversations
- Support more filetypes as message attachments 
- Ability to disable tweet conversation in the Twitter inbox
- Improved email confirmation flow for user accounts
- Improved email parsing
- Improved background job processing 
- New Language: Slovak(sk)

Thanks to @arabakevin, @heroic, @iprithvitharun, @PatelN123, @Anisha-art for the contributions

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v2.1.0'>v2.1.0</a>.</em>

**Comment by joaopaulospp** [comment]
Tem um guia de instalação ?

**Reply by sojan-official** [reply]
https://www.chatwoot.com/docs/self-hosted
